### DISCUSSION TITLE
Multiple projects at the same time

### DISCUSSION THREAD

**Discussion by roke22** [sfx101/docker-stacks#70] [Ideas]
Hello,
Sometimes i need to run two projects at the same time because one of them use a Rest API to get some data or do some jobs. What about make the NGINX + PHP with multiple domains?
Is ther any chance to do with the custom stack? Where can we look the types of the settings so we can ask for several folders in the customize stacks?

Thank you.

**Comment by nabad600** [comment]
Hello @roke22, thank you for using DECK, we understand your issue, you can create multiple stacks/containers for each project. 
For example, we are creating 2 instances (i.e. project1 and project2) and one DB (i.e. common-db) and all instance/stacks/containers are interconnected. Project1 container connecting to DB for fetching all the data and project2 container connecting project1 container through reset API. and all the connections are working properly. below mentioned snapshots for reference

![image](https://user-images.githubusercontent.com/67466982/208928109-5d06d0b6-f9ba-4e81-b69b-6f26c748eaab.png)
### structure

![image](https://user-images.githubusercontent.com/67466982/208928359-93c655fc-b6a4-4988-a07f-1fdb0823af45.png)
### Database structure

![image](https://user-images.githubusercontent.com/67466982/208928504-0333de61-4b50-49a1-a64c-9ac3bad077db.png)
### Endpoint to get data from DB

![image](https://user-images.githubusercontent.com/67466982/208928599-3407c6cb-993b-4110-9f1c-351849ac30fd.png)
### JSON file, that exists in the project (instance) directory, which will be called from another instance (project2).

![image](https://user-images.githubusercontent.com/67466982/208928690-c4f03526-5cd9-4b00-af84-40b974412d71.png)
### Call from project2 instance to get the json data from project1 instance

![image](https://user-images.githubusercontent.com/67466982/208929142-a7943ca3-a6d9-4933-bb1c-c865153078f5.png)
### project2 sample curl code

Thank you for your input. We'd be glad to assist.
